service-loadbalancer/service_loadbalancer.go

```diff
@@ -167,6 +170,12 @@ type service struct {
 	// The name of the cookie is SERVERID
 	// This only can be used in http services
 	CookieStickySession bool
+
+	// URLs are the paths used to route the traffic to the backend, like "<namespace>/<svc name>:<port>".
```

Comment:
suggest a rewording:
// URLs is a list of paths used to route traffic to the set of endpoints associated with a Service.
// Each URL takes the form: `<namespace>/<svc name>:<port>`

Add the comment about omissiong of the default namespace and port 80 above the implementation of getLBRules, since different rulegetters might do different things.